cropperjs/index.d.ts

```diff
@@ -4,6 +4,10 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
+declare const Cropper: typeof cropperjs.Cropper;
+export = Cropper;
+export as namespace cropperjs;
```

Comment:
This module exposes itself from the global `Cropper`, so this should be `export as namespace Cropper`